internal/handler/cmd_query.go

```diff
@@ -19,9 +19,10 @@ import (
 	"fmt"
 	"strings"
 
-	"github.com/FerretDB/FerretDB/internal/handler/common"
+	"go.uber.org/zap"
+
+	"github.com/FerretDB/FerretDB/internal/bson"
```

Comment:
@AlekSi we have a linter checking that `bson` namespace should not be imported, is `types` still preferred over `bson` in handler package?

Reply:
Actually I kept it to use types to avoid too many changes.